agent/engine/docker_task_engine_test.go

```diff
@@ -975,6 +984,9 @@ func TestCapabilities(t *testing.T) {
 		"com.amazonaws.ecs.capability.logging-driver.syslog",
 		"com.amazonaws.ecs.capability.selinux",
 		"com.amazonaws.ecs.capability.apparmor",
+		"com.amazonaws.ecs.capability.cni-plugin-bridge-1.0.0",
```

Comment:
Are these the names we want to use?  I think the newer ones we've been looking at start with just the `ecs.` prefix (so maybe `ecs.capability.cni-plugin-bridge`.  It might also be useful to start putting the version in the value of the attribute rather than the name ("1.0.0").